client/components/common/Header.jsx

```diff
@@ -1,38 +1,52 @@
 import React from "react";
 import { Link } from "react-router-dom";
-import { PropTypes } from 'prop-types';
-import { connect } from 'react-redux';
+import { PropTypes } from "prop-types";
+import { connect } from "react-redux";
 //action
-import { logout } from '../../actions/auth';
+import { logout } from "../../actions/auth";
 
 class Header extends React.Component {
   logout(e) {
```

Comment:
`e` may not be a very descriptive name for your parameter. you could check out this resource 

https://api.jquery.com/event.preventdefault/